client/landing/gutenboarding/controller.tsx

```diff
@@ -19,6 +20,10 @@ export const redirectIfNotEnabled: PageJS.Callback = ( context, next ) => {
 };
 
 export const main: PageJS.Callback = ( context, next ) => {
-	context.layout = <Gutenboard />;
+	context.layout = (
+		<Router>
+			<Gutenboard />
+		</Router>
+	);
```

Comment:
Seeing `<Router />` here while we're still using `page.js` in `./index.ts` is a bit confusing :thinking: Won't they collide? Can we ditch `page.js` in Gutenboarding?

Reply:
> Seeing `<Router />` here while we're still using `page.js` in `./index.ts` is a bit confusing 🤔 
> Won't they collide?

Agreed. I found that the did with the browser router. That's why I was using the hash router here.

> Can we ditch `page.js` in Gutenboarding?

I was under the assumption (that should be challenged) that page.js would be required to get us into the landing page route. Can our entrypoint work entirely outside page.js?

Reply:
> > Seeing `<Router />` here while we're still using `page.js` in `./index.ts` is a bit confusing thinking
> > Won't they collide?
> 
> Agreed. I found that the did with the browser router. That's why I was using the hash router here.
> 
> > Can we ditch `page.js` in Gutenboarding?
> 
> I was under the assumption (that should be challenged) that page.js would be required to get us into the landing page route. Can our entrypoint work entirely outside page.js?

I think so, yeah. (I think the current state of this PR, with my commit from last night pushed, should do so already.)

Reply:
Thinking about this a bit more, I think you're right. We'd replace the body of `window.AppBoot` here with our own boot function:

https://github.com/Automattic/wp-calypso/blob/dc7b6618d24e48c126de41a974bee67fc1a36667/client/landing/gutenboarding/index.ts#L19-L29

That should allow us to use the `BrowserRouter` without issue 👍

Reply:
Oh, I've arrived at the conclusion you already pushed! 🙌 😅